doc/cascadia/profiles.schema.json

```diff
@@ -1184,8 +1184,9 @@
               "pattern": "focusPane"
             },
             "id": {
-              "type": "string",
-              "default": "",
+              "type": "number",
```

Comment:
Nice find! Although `"integer"` would be a better fit than `"number"` here, wouldn't it?

Reply:
I didn't know there was `integer`, I saw `dropdownDuration` was a `number` with a minimum of 0, so I did the same.
So in that case, this means `dropdownDuration` is also wrong.